lib/main.js

```diff
@@ -18,8 +18,7 @@ export default {
 
     subscriptions.add(atom.config.observe('linter-puppet-lint', (settings) => {
       allSettings = settings;
-      executablePath = settings.executablePath;
-      errorLevel = settings.errorLevel;
+      ({ executablePath, errorLevel } = settings);
```

Comment:
@mschuchard Here's the proper syntax for that 😉. You can find details at [Destructuring assignment without declaration](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Assignment_without_declaration).

Reply:
Thanks for the tip. I tried:

```javascript
{ errorLevel } = settings;
```

based on the doc, and now I see how to actually do it.

Reply:
I didn't know about that syntax either till I ran into the same issue while updating `linter-eslint`, I'm surprised you don't see it around more often with how useful it is.